velox/functions/lib/tests/Re2FunctionsTest.cpp

```diff
@@ -640,6 +698,34 @@ TEST_F(Re2FunctionsTest, likePatternFixed) {
   testLike(input, input, true);
 }
 
+TEST_F(Re2FunctionsTest, likePatternRelaxedFixed) {
+  testLike("_ab_cde_", "_ab_cde_", true);
+  testLike("xabxcdex", "_ab_cde_", true);
+  testLike("xacxcdex", "_ab_cde_", false);
+
+  testLike("ABCDEx", "ABCDE_", true);
+  testLike("ABCDEy", "ABCDE_", true);
+  testLike("ABCDCy", "ABCDE_", false);
+  testLike("abcdey", "ABCDE_", false);
+
+  testLike("fblearner_global", "fblearner_", false);
```

Comment:
These test cases use Meta-specific terms and come as a surprise. Let's use something similar to previous test cases. 

testLike("ABCDE_", "ABCDE_", true);
testLike("ABCDE__", "ABCDE_", false);
testLike("ABCDE_ZZ", "ABCDE_", false);